client/src/main/java/com/paypal/selion/internal/platform/grid/MobileTestSession.java

```diff
@@ -40,6 +40,8 @@
     private String appName;
     private final String appLocation;
     private String device = "iphone";
+    private final String IPHONE = "iphone";
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [This final field could be made static](https://www.codacy.com/app/SeLion/paypal-SeLion/file/1529805667/issues/source?bid=2928425&fileBranchId=2989551#l43)